packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -43,15 +43,17 @@ You can also [configure a custom email notification domain](#about-custom-email-
 
 ### Caveats
 
-* Most custom domains are configured using one of your [subdomains](https://en.wikipedia.org/wiki/Subdomain). However, there are no restrictions on using any other domain such as your root domain, if that fits your use case for your sign-in page taking over your main website.
+* You must provide a [subdomain](https://en.wikipedia.org/wiki/Subdomain) for customizing the Okta URL domain. Okta doesn't support the use of a root domain name.
+
+* Any DNS TXT and CNAME record names and values included in your domain configuration must be resolvable and contain the values provided by Okta. You can validate these names and values with a DNS query tool, such as [dig](https://en.wikipedia.org/wiki/Dig_(command)).
```

Comment:
Should DNS TXT and CNAME have tics? (code-format?)
Also, not sure about linking to wikipedia? Is there another official link for this command?